compiler/z/CMakeLists.txt

```diff
@@ -57,7 +57,6 @@ compiler_library(z
 	${CMAKE_CURRENT_LIST_DIR}/codegen/OMRCodeGenerator.cpp
```

Comment:
Could you update the copyright header in this file please?

Reply:
Fixed in [force-push](https://github.com/eclipse/omr/compare/aa4d2cf9e7c421a5ea39e9f8d701fd14e58ddbca..0b3376f0ba6f553545daf6908b91dc0a9729d50e).